dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -256,18 +257,21 @@ private boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry) {
 
       if (!isClassAccessibleFrom(targetJavaClass, sourceClassName)) {
         return Optional.of(
-            new SymbolProblem(symbol, ErrorType.INACCESSIBLE_CLASS, containingClassFile));
+            new LinkageProblem(
```

Comment:
github won't let me put this comment on line 254 but classFileLocation --> classPathEntry

containingClassFile should probably be targetClassFile